config/prometheus/monitor.yaml

```diff
@@ -1,16 +1,20 @@
-
 # Prometheus Monitor Service (Metrics)
 apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
   labels:
     control-plane: controller-manager
-  name: controller-manager-metrics-monitor
-  namespace: system
+  name: capsule-monitor
+  namespace: capsule-system
```

Comment:
`Namespace` must remain to `system` since `kustomize` will change it at installation time.